src/commands/hsetxx.json

```diff
@@ -0,0 +1,69 @@
+{
+    "HSETXX": {
+        "summary": "Sets the value of a field in a hash only when the field exist.",
```

Comment:
Typo? Or did I misunderstand the command?

```suggestion
        "summary": "Sets the value of a field in a hash only when the key exist.",
```

Reply:
My bad

Reply:
I think correct will be as "Sets the value of a field in a hash only when the key and field exist"
@zuiderkwast

Reply:
Ah! OK. Then I misunderstood the command and also HSETNX. I thought that it is only about if the *key* exists.